compiler/damlc/tests/daml-test-files/RecordDotUpdates.daml

```diff
@@ -50,3 +50,15 @@ y = Outer {inner = x, text = "world", int = 21}
 y' = let text = "world" in let int = 12 in y {text, inner.text = text, int, inner.int = int}
 
 z = let Outer {..} = y in Outer {..}
+
+data One = One with two : Two
```

Comment:
maybe also a good idea to add a test that uses whatever the magic name is we use on the GHC side given how much trouble we had to go through to fix that :)

Reply:
That's a good idea. The magic name is 'arg' now.